README.md

```diff
@@ -1,23 +1,41 @@
-# fotogalleri
+# Fotogalleri - FAFA
 
 <p align="center">
     <img
-        src="https://raw.githubusercontent.com/Teknologforeningen/fotogalleri/master/.github/logo_black.svg"
+        src="https://raw.githubusercontent.com/Teknologforeningen/fotogalleri/master/.github/logo_black.svg?sanitize=true"
```

Comment:
Good to mention here that Github by defaults sanitizes resources stored under `raw`, so to ensure that the image is displayed correctly we have to pass `?sanitize=true` for it so that the SVG element is displayed correctly.

https://stackoverflow.com/a/16462143